app/src/main/java/com/xmartlabs/taskloans/ui/screens/dashboard/TeamAdapter.kt

```diff
@@ -0,0 +1,44 @@
+package com.xmartlabs.taskloans.ui.screens.dashboard
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.xmartlabs.taskloans.databinding.ListItemTeamBinding
+
+class TeamAdapter : ListAdapter<String, TeamAdapter.TeamHolder>(DiffCallback()) {
```

Comment:
I'd avoid string, string is not safe, if you want to use only strings I would use an inline class